src/Juvix/Extra/Paths/Base.hs

```diff
@@ -11,6 +11,9 @@ assetsDirQ = FE.makeRelativeToProject "assets" >>= FE.embedDir
 cssDirQ :: Q Exp
 cssDirQ = FE.makeRelativeToProject "assets/css" >>= FE.embedDir
 
+juvixStyQ :: Q Exp
+juvixStyQ = FE.makeRelativeToProject "latex/juvix.sty" >>= FE.embedFileRelative
```

Comment:
Suggestion: Use `include/latex/juvix.sty` for the file. The `include/` directory in the repository is intended for extra things to be included in the binary.

Reply:
done